x-pack/test/reporting/functional/reporting.js

```diff
@@ -198,46 +173,32 @@ export default function ({ getService, getPageObjects }) {
         });
 
         it('becomes available when saved', async () => {
-          await PageObjects.dashboard.saveDashboard('mypngdash');
+          await PageObjects.dashboard.saveDashboard('My PNG Dash');
           await PageObjects.reporting.openPngReportingPanel();
           await expectEnabledGenerateReportButton();
         });
       });
 
-      // TODO Re-enable the tests after removing Phantom:
-      // https://github.com/elastic/kibana/issues/21485
-      describe.skip('Preserve Layout', () => {
+      describe('Preserve Layout', () => {
```

Comment:
Preserve Layout should be moved away from, as a concept of the Reporting Service. 

It is a mode of the Kibana App dashboard integration.